build-parent/pom.xml

```diff
@@ -83,6 +83,13 @@
         <opensearch.image>opensearchproject/opensearch:${opensearch-server.version}</opensearch.image>
         <opensearch.protocol>http</opensearch.protocol>
 
+        <!-- Database images for JDBC/Reactive/Hibernate tests -->
+        <postgres.image>postgres:14.1</postgres.image>
+        <mariadb.image>mariadb:10.4</mariadb.image>
+        <db2.image>ibmcom/db2:11.5.5.0</db2.image>
+        <mssqldb.image>mcr.microsoft.com/mssql/server:2019-CU8-ubuntu-16.04</mssqldb.image>
```

Comment:
Why `mssqldb` instead of `mssql`?

Reply:
I picked the name used originally in `integration-tests/jpa-mssql/pom.xml`.

Reply:
OK, I adjusted it to be more consistent.